tests/templates/kuttl/kerberos-ad/kinit-client.yaml

```diff
@@ -21,11 +22,11 @@ spec:
               echo listing keytab contents
               klist -k /stackable/krb/keytab -teKC
               echo kiniting node
-              kinit -kt /stackable/krb/keytab -p HTTP/$NODE_NAME
+              kinit -kt /stackable/krb/keytab -p HTTP/\$NODE_NAME
```

Comment:
This also seems likely to trip someone up who is looking at the test files as an example.

Reply:
True. Strictly speaking, this is not a YAML file but a templated file meant to be evaluated by a shell. So the file name should actually be suffixed by something, e.g. `kinit-client.yaml.with-envs`, to make it clear that it must be preprocessed before it can be applied.

We have to discuss it in our team if we want to add `yq` to the test driver or if we want to use `eval` or `kubectl apply -f - <<EOF` or anything else.